Library/Homebrew/test/test_os_mac_dependency_collector.rb

```diff
@@ -19,4 +23,16 @@ def test_ld64_dep_leopard_or_newer
     MacOS.stubs(:version).returns(MacOS::Version.new("10.5"))
     assert_nil @d.build(:ld64)
   end
+
+  def test_ant_dep_mavericks_or_newer
+    MacOS.stubs(:version).returns(MacOS::Version.new("10.9"))
+    @d.add ant: :build
+    assert_equal find_dependency("ant"), Dependency.new("ant", [:build])
+  end
+
+  def test_ant_dep_pre_mavericks
+    MacOS.stubs(:version).returns(MacOS::Version.new("10.7"))
+    @d.add ant: :build
+    assert_nil find_dependency("ant")
+  end
```

Comment:
I had another test when I first added this to Linuxbrew. Something very similar should do.